modules/agar/src/test/ts/browser/datatransfer/DataTransferItemListTest.ts

```diff
@@ -24,7 +24,7 @@ UnitTest.test('DataTransferItemListTest', () => {
 
     assert.throwsError(() => {
       items.add('123', 'text/plain');
-    }, `Failed to execute 'add' on 'DataTransferItemList': An item already exists for type 'text/plain'.`);
+    }, 'Failed to execute \'add\' on \'DataTransferItemList\': An item already exists for type \'text/plain\'.');
```

Comment:
This seems like a configuration that needs to be tweaked, as there's no reason to have to escape all these when we can use backtick/template strings.

Reply:
or switch to double quotes for those strings

Reply:
Forgot to mention everything else looks good, I just don't think we should be forced to escape every apostrophe. It's also possible that this might be a place where regular quotes instead of single quotes should be used?

Reply:
We may need to enable the `avoidEscape` setting in the rules: https://eslint.org/docs/rules/quotes#avoidescape

Reply:
`"single", { "avoidEscape": true }` looks good to me

Reply:
Will do.

Reply:
So ended up using backticks instead of double quotes as it seemed to be the preference of the group in enchiladacity.